### DISCUSSION TITLE
2022-11-16 Weekly Meeting 82 Agenda

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/omigroup#317] [Weekly Meetings]
Weekly Meeting 82 is happening on Wednesday November 16, 2022 from 9:00 AM PST(17:00 UTC) - 10:00 AM PST (18:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat. 

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda
- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers

### Working Group Updates:
- [MSF delegates](https://github.com/omigroup/MSF-Delegates) (setting up the repo)
- [UX Research](https://github.com/omigroup/ux-research) 
- [Scripting group](https://github.com/omigroup/omi-scripting-group)
  - OMI developer hour* 
- [glTF Extensions Group](https://github.com/omigroup/gltf-extensions)
  - Progress on OMI_ref and authoring extensions
- [Media Group](https://github.com/omigroup/media)
- [M3 avatar interop](https://github.com/M3-org/avatar-interop#calendar)
- [OMI Website update](https://github.com/omigroup/website)
- any others?

### Discussions: 
TBD! 

#### Next week: 
TBD!

### Future demo themes: 
- `OMI-audio-emitter` exploration and demo 
- `OMI-collider` exploration and demo 
- `Generative and AI tooling`
- Any other suggestions?

**Comment by mrmetaverse** [comment]
# Ice-breaker / M3 Avatar Interop Group 

Jin shared [this tweet](https://twitter.com/m3org/status/1592920552238612481) from the Avatar Interop groups last session together. They spent half of their meeting in one place, and the other half in VR Chat. 

![image](https://user-images.githubusercontent.com/63426722/202245893-fa9ca81f-2bee-481f-93ca-8ea8d531c950.png)

- Maybe we (OMI) should start doing more of this ourselves too. **More trips in VR**. 
- Did M3TV premier on Friday 
- We want to do an OMI premier as well. 
- Could align with media group, we can cut up 30 min of content into different formats. Interviews, panels, commercials, music videos, etc. Inspired by MTV. 
- Content relevant to Metaverse Makers. Started with a [hackmd](https://hackmd.io/@XR/vprod/https%3A%2F%2Fhackmd.io%2F%40XR%2Fvideos) list. 
- Agenda: 
- ![image](https://user-images.githubusercontent.com/63426722/202252843-4bedfdff-ca60-494b-9ffc-9bacd03a2079.png)
- Jin also made a snapshot of Voxels wearables. This could be useful for [asset interoperability exploration as well](https://twitter.com/dankvr/status/1592341122181132288) (@Elirudite :eyes: )
- [Smooth voxels](https://svox.glitch.me/) is a fun tool working with voxels. 
- passed a DAO proposal to convert live2d avatars to 3d VRMs. This passed, and so it will supplement some funding for artists to create the traits for people who create tools for the avatars. And to do auto-optimization work. We all have open-source code bases for these parts of the toolchain. 
- If you know anyone with VROID experience please put me in touch. 
- Also still creating and exploring virtual spaces and "war rooms" to work together. https://hackmd.io/@XR/anata/https%3A%2F%2Fhyperfy.io%2Fanata
- ![image](https://user-images.githubusercontent.com/63426722/202253656-cbdfd713-29eb-42a7-b480-ea135d812ff2.png)

**Comment by mrmetaverse** [comment]
# Next Demo event: Using OMI Protocols? 

What if we demo OMI protocol usage over the next two weeks? 

@robertlong @antpb Any interest in demoing usage of our protocols? Also, does anyone come to mind that we could invite to a future Wednesday?

**Comment by mrmetaverse** [comment]
# MSF Delegates 
- notes [11/16/22](https://github.com/omigroup/MSF-Delegates/discussions/35)
- considering fee structure for anyone who wants a board seat, to fund MSF 
  - varying fees, do you want a vote or not? 
  - A lot of discussion and debate around inclusion and equity 
  - No resolution, just more discussion of what would be feasible. 
  - Ended with a suggested poll. 
- MSF is forming a non-profit in Deleware 
  - will still be free for some
  - but will have a paid process too 
- the intent is to have MSF form its own org to step away from Khronos. 
- estimated $125k to cover budget

**Comment by mrmetaverse** [comment]
# UX - Research 
Notes! https://github.com/omigroup/ux-research/discussions/15

Spread the word!

**Comment by mrmetaverse** [comment]
# gLTF 
- Last meeting: https://github.com/omigroup/gltf-extensions/discussions/116 
- OMI Link is new! 
- Next meeting is tomorrow! [Agenda](https://github.com/omigroup/gltf-extensions/discussions/117)

**Comment by mrmetaverse** [comment]
# Media 

- We are seeking someone to manage our mastadon account. 
- notes from last week: https://github.com/omigroup/omigroup/discussions/313
- There is a sign-up form for those who are interested in speaking on OMI behalf.
- We are seeking new streaming opportunities too. 
- Next meeting is next week!
